website/docs/r/databricks_virtual_network_peering.html.markdown

```diff
@@ -0,0 +1,107 @@
+  name                = "databricks-vnet-peer"
+  resource_group_name = azurerm_resource_group.example.name
+  workspace_id        = azurerm_databricks_workspace.example.id
+
+  remote_address_space_prefixes = azurerm_virtual_network.remote.address_space
+  remote_virtual_network_id     = azurerm_virtual_network.remote.id
+  allow_virtual_network_access  = true
+}
+
+resource "azurerm_virtual_network_peering" "remote" {
+  name                         = "peer-to-databricks"
+  resource_group_name          = azurerm_resource_group.example.name
+  virtual_network_name         = azurerm_virtual_network.remote.name
+  remote_virtual_network_id    = azurerm_databricks_virtual_network_peering.example.virtual_network_id
+  allow_virtual_network_access = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Databricks Virtual Network Peering resource. Possible valid values must begin with a letter or number, end with a letter, number or underscore, and may contain only letters, numbers, underscores, periods, or hyphens and must be between 1 and 80 characters in length. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group in which the Databricks Virtual Network Peering should exist. Changing this forces a new resource to be created.
+
+* `workspace_id` - (Required) The Id of the Databricks Workspace that this Databricks Virtual Network Peering is bound. Changing this forces a new resource to be created.
+
+* `address_space_prefixes` - (Required) A list of address blocks reserved for this virtual network in CIDR notation. Changing this forces a new resource to be created.
+
+* `remote_address_space_prefixes` - (Required) A list of address blocks reserved for the remote virtual network in CIDR notation. Changing this forces a new resource to be created.
+
+* `remote_virtual_network_id` - (Required) The Id of the remote virtual network. Changing this forces a new resource to be created.
+
+~> **NOTE:** The remote virtual network should be in the same region as the databricks workspace. Please see the [product documentation](https://learn.microsoft.com/azure/databricks/administration-guide/cloud-configurations/azure/vnet-peering) for more information.
+
+* `allow_virtual_network_access` - (Optional) Can the VMs in the local virtual network space access the VMs in the remote virtual network space? Defaults to `true`.
+
+* `allow_forwarded_traffic` - (Optional) Can the forwarded traffic from the VMs in the local virtual network be forwarded to the remote virtual network? Defaults to `false`.
+
+* `allow_gateway_transit` - (Optional) Can the gateway links be used in the remote virtual network to link to the Databricks virtual network? Defaults to `false`.
+
+* `use_remote_gateways` - (Optional) Can remote gateways be used on the Databricks virtual network? Defaults to `false`.
```

Comment:
these defaults should be defined in the Schema rather than the field being Computed

Reply:
Fixed.